### DISCUSSION TITLE
Should I just move my base directory to Dropbox?

### DISCUSSION THREAD

**Discussion by peterjduffy** [remotely-save/remotely-save#633] [Q&A]
Hi there, newbie question. I have a MacBook and iPhone. My obsidian files are stored in iCloud, then replicated to Dropbox via remotely save. I always get a 'waiting for iCloud' message when opening a vault on my phone. With remotely save, should I just move the base folder to Dropbox? 

thanks!

**Comment by fyears** [comment]
> I always get a 'waiting for iCloud' message when opening a vault on my phone. 

I don't know why, and I **think** Remotely Save doesn't involve with iCloud.

> With remotely save, should I just move the base folder to Dropbox?

No. You base folder is on your mobile (or pc). Remotely Save just "copy" your files by treating Dropbox as a "file server".
